mautrix_telegram/portal/metadata.py

```diff
@@ -450,7 +450,7 @@ def _get_base_power_levels(self, levels: PowerLevelStateEventContent = None,
             levels.kick = overrides.get("kick", 50)
             levels.redact = overrides.get("redact", 50)
             levels.invite = overrides.get("invite", 50 if dbr.invite_users else 0)
-            levels.events[EventType.ROOM_ENCRYPTION] = 99
+            levels.events[EventType.ROOM_ENCRYPTION] = overrides.get("encryption", 50 if self.matrix.e2ee else 99)
```

Comment:
Event level overrides are already applied below on lines 469-470

```suggestion
            levels.events[EventType.ROOM_ENCRYPTION] = 50 if self.matrix.e2ee else 99
```

Reply:
Oops, I completely overlooked that section of the code, added your suggestion to the PR